grpc-server/src/main/java/com/netflix/conductor/grpc/server/service/MetadataServiceImpl.java

```diff
@@ -70,6 +79,15 @@ public void getWorkflow(MetadataServicePb.GetWorkflowRequest req, StreamObserver
         }
     }
 
+    @Override
+    public void getTasks(MetadataServicePb.GetTasksRequest req, StreamObserver<MetadataServicePb.GetTasksResponse > response) {
```

Comment:
**Suggestion**: can we use `getTaskDefs` or `getTaskDefinitions`?
Similar concept stated with workflows but with task definitions: A task is a runtime object that contains a task definition.